docker-compose.yml

```diff
@@ -94,6 +92,7 @@ services:
       - app/http.php
     environment:
       - _APP_ENV
+      - _APP_EDITION
```

Comment:
lets make sure we also add this variable in the cloud repo

Reply:
Yep done :+1: